src/Domain/CoordinateMaps/TimeDependent/Skew.cpp

```diff
@@ -80,8 +80,9 @@ std::optional<std::array<double, 3>> Skew::inverse(
   }
```

Comment:
Just to make sure I'm understanding correctly: The sign being changed here is the sign of the FoT, correct?

Reply:
Yes it's the sign of the FoT which has changed, which propagates out the overall sign of the sum `tan(F_y) * (y-y_C) + tan(F_z) * (z-z_C)` because of the `tan`s